### PR TITLE
Add ReadOnlyPage to the list of mitigations in scope for the Exploit Mitigation bounty

### PR DESCRIPTION
**Pull request by tomrittervg**

### CODE REVIEW

**Comment by tomrittervg** [comment]
@dveditz Tagging you for review

**Review by dveditz — APPROVED** [review]
LGTM

**Comment by tomrittervg** [comment]
@pmac Tagging you!

**Comment by tomrittervg** [comment]
I don't know why but I cannot request Github reviewers.

File: bedrock/security/templates/security/client-bug-bounty.html [hunk 1]
```diff
@@ -136,6 +136,7 @@ <h2 id="exploit-mitigation-bounty">Exploit Mitigation Bug Bounty</h2>
...
+    <li>We have a <a href="https://searchfox.org/mozilla-central/search?q=ReadOnlyPage&path=">ReadOnlyPage</a> <a href="https://searchfox.org/mozilla-central/rev/3fa5cc437a4937c621ea068ba5dc246f75831633/js/xpconnect/src/nsXPConnect.cpp#1018">implementation</a> which is intended to prevent Data-Only attacks. A bypass would be reliably exploiting a race condition when the memory region is unprotected, finding a bug such that the memory region is left unprotected, or identifying a way around the mitigation to effect a data-only attack on the protected values.
```
Comment (pmac) [inline bedrock/security/templates/security/client-bug-bounty.html:139]: Missing the `</li>`

Comment (tomrittervg) [reply bedrock/security/templates/security/client-bug-bounty.html]: done

**Comment by pmac** [comment]
> I don't know why but I cannot request Github reviewers.

GitHub only allows people with repo write perms to do that ¯\\\_(ツ)\_/¯

**Review by pmac — APPROVED** [review]
🥦

**Pull request merged**